docs/ruleset_functions/is_hvac_sys_fluid_loop_type_CHW(hvac.id).md

```diff
@@ -0,0 +1,26 @@
+# is_hvac_sys_fluid_loop_type_CHW  
+
```

Comment:
should the name of this function be is_hvac_sys_fluid_loop_type_Purchased_CHW  ?